invesalius/gui/task_navigator.py

```diff
@@ -1337,6 +1350,10 @@ def __init__(self, parent, navigation, tracker, icp):
         self.navigation = navigation
         self.tracker = tracker
         self.icp = icp
+        if has_mTMS:
```

Comment:
Wouldn't it be better to just assign self. mTMS already to the mTMS call done in lines 34-36 above? Alternatively, the try-except could be done inside the mTMS() method call, to make it more clean, and return True or False to the task_navigator. This would remove two "try-except" calls from the task_navigator

Reply:
TODO